packages/target-tools/src/murmurhash3.js

```diff
@@ -0,0 +1,123 @@
+/* eslint-disable */
+/*
+ *  The MurmurHash3 algorithm was created by Austin Appleby.  This JavaScript port was authored
+ *  by whitequark (based on Java port by Yonik Seeley) and is placed into the public domain.
+ *  The author hereby disclaims copyright to this source code.
+ *
+ *  This produces exactly the same hash values as the final C++ version of MurmurHash3 and
+ *  is thus suitable for producing the same hash values across platforms.
+ *
+ *  There are two versions of this hash implementation. First interprets the string as a
+ *  sequence of bytes, ignoring most significant byte of each codepoint. The second one
+ *  interprets the string as a UTF-16 codepoint sequence, and appends each 16-bit codepoint
+ *  to the hash independently. The latter mode was not written to be compatible with
+ *  any other implementation, but it should offer better performance for JavaScript-only
+ *  applications.
+ *
+ *  See http://github.com/whitequark/murmurhash3-js for future updates to this file.
+ */
+
+const MurmurHash3 = {
+  mul32: function (m, n) {
+    const nlo = n & 0xffff;
+    const nhi = n - nlo;
+    return (((nhi * m) | 0) + ((nlo * m) | 0)) | 0;
+  },
+
+  hashBytes: function (data, len, seed) {
```

Comment:
does it make sense to still have this in code, if we've already decided to only use hashString?

Reply:
I think it's fine to keep it.  It will be tree-shaken out of any bundles anyway (if it's not used).  So why not leave it in `target-tools` in case we want to use it at some point.